src/transparency/glue.rs

```diff
@@ -303,15 +306,15 @@ where
 impl<F, B> Future for HyperServerSvcFuture<F>
 where
     F: Future<Item=http::Response<B>>,
-    F::Error: fmt::Debug,
+    F::Error: Error + fmt::Debug + Send + Sync + 'static,
 {
     type Item = http::Response<BodyPayload<B>>;
     type Error = h2::Error;
 
     fn poll(&mut self) -> Poll<Self::Item, Self::Error> {
         let mut res = try_ready!(self.inner.poll().map_err(|e| {
-            debug!("h2 error: {:?}", e);
-            h2::Error::from(io::Error::from(io::ErrorKind::Other))
+            debug!("h2 error: {}", e);
```

Comment:
This log message looks funny... it's not actually an h2 error, right?

Reply:
Ah, you're right! Will fix.